lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.cpp

```diff
@@ -136,13 +136,14 @@ void PopulateSelection(
     }
 
     SdfPath usdPath = usdItem->prim().GetPath();
+    const int instanceIndex = usdItem->instanceIndex();
 
 #if !defined(USD_IMAGING_API_VERSION) || USD_IMAGING_API_VERSION < 11
     usdPath = sceneDelegate.ConvertCachePathToIndexPath(usdPath);
 #endif
 
     sceneDelegate.PopulateSelection(
-        HdSelection::HighlightModeSelect, usdPath, UsdImagingDelegate::ALL_INSTANCES, result);
+        HdSelection::HighlightModeSelect, usdPath, instanceIndex, result);
```

Comment:
This is the reason the current code selects all instances of the prototype. If a particular instance was selected, we want to be sure to populate the Hydra selection with the index of that instance.

Reply:
I can definitely see why the change is required, but I don't understand yet how selection highlight could work on exact instance before. What is the difference between point instancer and regular instancing (the one I see in treePatch.usda) when it comes to PopulateSelection()?